lib/ansible/modules/cloud/azure/azure_rm_containerregistry_facts.py

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2017 Zim Kalinowski, <zikalino@microsoft.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_containerregistry_facts
+version_added: "2.7"
+short_description: Get Registry facts.
+description:
+    - Get facts of Registry.
+
+options:
+    resource_group:
+        description:
+            - The name of the resource group to which the container registry belongs.
+        required: True
+    registry_name:
+        description:
+            - The name of the container registry.
+
+extends_documentation_fragment:
+    - azure
+
+author:
+    - "Zim Kalinowski (@zikalino)"
+
+'''
+
+EXAMPLES = '''
+  - name: Get instance of Registry
+    azure_rm_containerregistry_facts:
+      resource_group: resource_group_name
+      registry_name: registry_name
+
+  - name: List instances of Registry
+    azure_rm_containerregistry_facts:
+      resource_group: resource_group_name
+'''
+
+RETURN = '''
+registries:
+    description: A list of dict results where the key is the name of the Registry and the values are the facts for that Registry.
+    returned: always
+    type: complex
+    contains:
```

Comment:
Can we show username/password, webhook list, registry list, registry tag list and buildtask list in the facts?